src/parser/processor.ts

```diff
@@ -100,6 +100,7 @@ export interface Result {
     total: number;
     task?: {
       reward: number;
+      multiplier: number;
```

Comment:
@gentlementlegen Could you also update data structure example in the readme file?

Reply:
Updated at https://github.com/ubiquibot/conversation-rewards/pull/50/commits/47323297446ea368eda467e86ed684bdf1651c22